### DISCUSSION TITLE
v2raya该使用什么样的订阅链接？

### DISCUSSION THREAD

**Discussion by hulyoo** [v2rayA/v2rayA#1418] [Q&A] [answered]
我试了几个梯子，都无法导入，但是在v2rayN,clash和shadowrocket都能正常使用，求解决办法。

**Comment by hulyoo** [comment]
这是导入SSR订阅链接的日志信息：
`failed to resolve subscription address: Get "https://b0520.xn--wqr30o34q.xn--io0a7i/link/kYVCtKFtEL2ecVfO?sub=1": dial tcp: lookup b0520.xn--wqr30o34q.xn--io0a7i on 192.168.0.1:53: read udp 192.168.0.6:41213->192.168.0.1:53: i/o timeout`  
这是导入V2ray订阅链接的日志信息：
`failed to resolve subscription address: Get "https://b0520.xn--wqr30o34q.xn--io0a7i/link/kYVCtKFtEL2ecVfO?sub=3": dial tcp: lookup b0520.xn--wqr30o34q.xn--io0a7i on 192.168.0.1:53: read udp 192.168.0.6:37130->192.168.0.1:53: read: connection refused`

**Comment by hulyoo** [comment, marked as answer]
今天重新导入的时候成功了。。。。很抽象

**Reply by hulyoo** [reply]
准确的说是我在debian系统中配置了DNS之后，导入就成功了

**Reply by demoAlitalia** [reply]
DNS配置的多少呢？我这边也是同样的情况，无法导入v2rayN的正常节点。报错：connection reset by peer 

v2raya version:2.2.5.6
环境：debian 5.18.5
已安装v2ray和xray

服务正常可以启动，我去添加我在v2rayN的订阅节点过来，报错。connection reset by peer 
![error](https://github.com/v2rayA/v2rayA/assets/59221123/eb38bb76-37cd-4bab-ae51-9641e8a3cf74)

**Reply by demoAlitalia** [reply]
或者也会报 i/o timeout

**Reply by Philipflyg** [reply]
> 或者也会报 i/o timeout



> DNS配置的多少呢？我这边也是同样的情况，无法导入v2rayN的正常节点。报错：connection reset by peer
> 
> v2raya version:2.2.5.6 环境：debian 5.18.5 已安装v2ray和xray
> 
> 服务正常可以启动，我去添加我在v2rayN的订阅节点过来，报错。connection reset by peer ![error](https://private-user-images.githubusercontent.com/59221123/346955283-eb38bb76-37cd-4bab-ae51-9641e8a3cf74.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcyNDY1MjYsIm5iZiI6MTcyNzI0NjIyNiwicGF0aCI6Ii81OTIyMTEyMy8zNDY5NTUyODMtZWIzOGJiNzYtMzdjZC00YmFiLWFlNTEtOTY0MWU4YTNjZjc0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI1VDA2MzcwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJiMmVkMWEzYzhlNGU3NDg1OTE2MTQwZjljNzE3NGZlODEzY2MxZDhmZmFkZTk3NGQ4ZDYxOTUzNTFkODRhNWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.0kf0tO3sLmyNXWrEU1IOW7-ir8qnWnlItdm1AuXKSlc)

同样问题，请问有解决吗？

**Comment by Taeyeonzpx** [comment]
> 我试了几个梯子，都无法导入，但是在v2rayN,clash和shadowrocket都能正常使用，求解决办法。

一样的错 现在更新订阅有啥技巧吗?

**Reply by demoAlitalia** [reply]
已转 clash
